scripts/stac/imagery/collection.py

```diff
@@ -130,6 +131,19 @@ def add_capture_area(self, polygons: list[BaseGeometry], target: str, artifact_t
         if StacExtensions.file.value not in self.stac["stac_extensions"]:
             self.stac["stac_extensions"].append(StacExtensions.file.value)
 
+    def add_capture_dates(self, source_directory: str) -> None:
```

Comment:
nit: missing docstring comment

Reply:
If this is something we want to enforce, pylint can do that for us.